### DISCUSSION TITLE
How is this different from CrewAI?

### DISCUSSION THREAD

**Discussion by nyck33** [Holmeswww/AgentKit#24] [Q&A] [answered]
https://github.com/joaomdmoura/CrewAI

At a high level can you explian differences?

**Comment by Holmeswww** [comment, marked as answer]
Hi,

  The CrewAI seems to be like a startup platform, whereas AgentKit is a light-weight research project.

  Please be advised of the following difference from my perspective. (Note that this may not be complete and may be biased)

  Differences:
     1. CrewAI is has more integrations. AgentKit depends on minimal things. (CrewAI has more features presumably. AgentKit is more likely to work in the future regardless of dependencies)
     2. AgentKit allows for more fine-grained control for what you want to do for a single task with DAG. CrewAI is more about a bunch of agents having interesting interactions (it still can't achieve much with the GPT-4-xxx models, maybe GPT-5 will solve this?).
